packages/kit/test/apps/basics/src/hooks.js

```diff
@@ -18,6 +18,20 @@ export const handleError = ({ request, error }) => {
 		: {};
 	errors[request.url.pathname] = error.stack || error.message;
 	fs.writeFileSync('test/errors.json', JSON.stringify(errors));
+	console.error(error.message);
```

Comment:
I'm not sure we'd want to be logging to the console even in tests

Reply:
Prior to the recent switch to playwright there was no handleErrors hook in the tests. That's why I mimicked the default implementation (which is logging errors to console.error) as some tests were depending on the output to be exactly this way. But with the recent changes this is not needed anymore and I'll remove the console.errors

Reply:
Removed console.error from test